text/0000-interactive-app-creation.md

```diff
@@ -0,0 +1,142 @@
+
+This interactive workflow will also ignore any ~/.ember-cli settings that already exist. A separate RFC should be written that makes it possible for these to things to co-exist and respect the settings of the other.
+
+### Future possibilities
+An example of a future RFC that could be specific to addons, would be to add a `--supported-version` flag for addons and add that to the wizard: 
+
+- `What is the earliest version of Ember that you intend to support? (use arrow keys)`
+  - `> recommended (last two LTS)`
+  - `last LTS`
+  - `manually define a version number`
+
+### Implementation
+This section is not normative, but provided for extra context for folks who might be new to this idea in general. We could use something like [inquirer.js](https://github.com/SBoudrias/Inquirer.js) to implement this wizard, if the mechanism doesn't already exist within the ember-cli codebase. However, it should be explicitly noted that inquirer is only being used as an example of a library that could be used, and this RFC isn't explicitly defining that inquirer.js should be or will be used. 
+
+## How we teach this
+
+As this is a wholly new idea, it should be documented and added to the guides along with screenshots of the new workflow. Until that prototype exists, this section will largely remain empty. However, we intend to explain it similarly to Vue's documentation for a similar feature (https://cli.vuejs.org/guide/creating-a-project.html#vue-create).  
+
+## Drawbacks
+
+It could be considered a drawback to make this not the default for new Ember apps, even if an app name is defined. 
+
+The general drawback with wizards is that it can lead to a large number of combinations, which increases the risk for fragmentation. We believe that we have limited this risk by limiting the number of questions asked, but it is still a risk and should be identified as such.
+
+## Alternatives
+
+1. We could decide on a different list of questions. See the notes from the [strike team discussion on May 20th](https://github.com/ember-a11y/core-notes/blob/ember-a11y/ember-a11y/2020-05/may-20.md) to review the other possibilities considered.
+1. We could make this the default for new Ember apps and add it in the appropriate version as a change.
+1. We could have this workflow be opt-in only, via the `--interactive` flag (e.g., `ember new --interactive`). Using the `ember new` command would error and trigger the help workflow (which would be updated to include the `--interactive` flag).
+1. We could set the default language to US English (the default language of the Ember project) and remove the language question from the interactive workflow.
+1. We could set the default language to US English (the default language of the Ember project) and not have an interactive option at all.
+
+## Unresolved questions
+1. Do we have a good way to make this a good experience for developers who use assistive technology?
```

Comment:
How many Ember developers are using assistive technology? What problems to they have with Ember today?

If the underlying purpose here is to "reach" more developers? Are there other groups that are left out at the moment, for example developers that don't speak English very well? Perhaps translating the docs (or parts of them) to more languages would reach more developers?

Reply:
I don't know the total number but I am aware that there are _some_; after digging into this unresolved question a bit more, I am fairly confident that we can remove it. The answer is, "the terminal has to do most of the heavy lifting here" at least where development is concerned. 

For the Ember team, providing clear error messages and ensuring that our guides and API docs are accessible will be the best support we can provide.

Reply:
Sounds reasonable, to solve most accessibility concerns on other levels than Ember. There are many interesting projects and great progress at the [editor](https://github.com/microsoft/vscode/issues/88987), [terminal](https://github.com/tspivey/tdsr) and [os](https://www.apple.com/accessibility/mac/) level. But I think these things are mostly better to address outside Ember.

If this RFC isn't mainly about accessibility though (sounds like it's more for first experience), maybe the motivation text in the RFC could be tweaked accordingly.

> The underlying motivation for this RFC is the desire to improve the accessibility of new Ember apps as previously documented in Issue 595/Section 4. Specifically, it aims to prevent new Ember apps from failing legal requirements for accessibility conformance according to WCAG 2.1, the standard used by most countries around the world to inform digital accessibility requirements.